### PR TITLE
Document running a local version of lightning on the cloud

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
See title.

### Does your PR introduce any breaking changes? If yes, please list them.
None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [n/a] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -309,6 +309,17 @@ and the last true master commit is `ccc111` and your first commit is `mmm222`.
   git push -f
   ```
 
+#### How to run an app on the cloud with lightning that you're making changes
```
Comment (nohalon) [inline .github/CONTRIBUTING.md:312]:
```suggestion
#### How to run an app on the cloud with a local version of lightning
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -309,6 +309,17 @@ and the last true master commit is `ccc111` and your first commit is `mmm222`.
...
+
```
Comment (nohalon) [inline .github/CONTRIBUTING.md:313]: can you add some motivation here for why you would want to use this

Comment (akihironitta) [reply .github/CONTRIBUTING.md:313]: PTAL!

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -309,6 +309,17 @@ and the last true master commit is `ccc111` and your first commit is `mmm222`.
   git push -f
   ```
 
+#### How to run an app on the cloud with a local version of lightning
+
+The lightning cloud uses the latest release by default. However, you might want to run your app with lightning that you have made changes to in your directory. To use your local version of lightning on the cloud, set the following environment variable:
+
+```bash
+export PACKAGE_LIGHTNING=1  # <- this is the magic to use local version!
```
Comment (Borda) [inline .github/CONTRIBUTING.md:317]:
```suggestion
export PACKAGE_LIGHTNING=1  # <- this is the magic to use your version (not mainstream PyPI)!
```

**Comment by akihironitta** [comment]
> do you still need to have it install with `-e` as `pip install -e .`

The use case of this env var I see is for developers who make changes to the lightning codebase, so I put `-e` in the guide there. Feel free to make any changes if you have another idea :)

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (Borda) [reply .github/CONTRIBUTING.md:313]: @akihironitta added? :otter:

Comment (akihironitta) [reply .github/CONTRIBUTING.md:313]: Yes! I belive so :)

**Review by nohalon — APPROVED** [review]

**Pull request merged**